requirements.txt

```diff
@@ -1,3 +1,5 @@
+# Python libraries used by Cozy.  Install by running:
+#   pip3 install -r requirements.txt
```

Comment:
I have mixed feelings about this comment. On the one hand, it satisfies a desire to have a description for every file. On the other hand,

 - it duplicates information in the readme
 - requirements.txt is an _incredibly_ common pattern in Python projects

(Related: I am actually very surprised to learn that [`setuptools` supports comments in requirements passed to `setup`](https://setuptools.readthedocs.io/en/stable/setuptools.html#declaring-dependencies).)

Reply:
I didn't know about requirements.txt (this shows how long it has been since I did serious Python programming).  People reading the Cozy implementation may not be familiar with Python.  But if you feel strongly, we can remove it.